internal/core/constants.go

```diff
@@ -55,13 +74,67 @@ func (it InvariantType) String() string {
 	case TxCaller:
 		return "tx_caller"
 
+	case BalanceEnforcement:
+		return "balance_enforcement"
+
 	default:
 		return "unknown"
 	}
 }
 
+// StringToInvariantType ... Converts a string to an invariant type
+func StringToInvariantType(stringType string) InvariantType {
+	switch stringType {
+	case "example":
+		return ExampleInv
+
+	case "tx_caller":
+		return TxCaller
+
+	case "balance_enforcement":
+		return BalanceEnforcement
+
+	default:
+		return InvariantType(0)
+	}
+}
+
+// AlertDestination ... The destination for an alert
+type AlertDestination uint8
+
+const (
+	Slack        AlertDestination = iota + 1
+	CounterParty                  // 2
+)
+
+func (ad AlertDestination) String() string {
+	switch ad {
+	case Slack:
+		return "slack"
+	case CounterParty:
+		return "counterparty"
+	default:
+		return "unknown"
+	}
+}
```

Comment:
Bit confused as to what a counterparty is. Assuming it just means anything but slack?